File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {

        unordered_set <int> arr;
        for (int num : nums){
            if (arr.count(num)){
                return true;
            }
            arr.insert(num);
        }
        return false;
        
    }
};

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        k = len(nums)//3
        count = {}
        res = []
        
        for n in nums:
            if n in res:
                continue
            count[n] = count.get(n, 0) + 1
            if count[n] > k:
                res.append(n)
        
        return res
        